acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ThreadBesuNodeRunner.java

```diff
@@ -133,13 +144,20 @@ public void startNode(final BesuNode node) {
             .withMetricsSystem(metricsSystem)
             .build();
 
+    final SecurityModule securityModule =
+        securityModuleService
+            .getByName("bouncycastle")
```

Comment:
is there a precedent for the case of this identifier? eg bouncyCastle, BouncyCastle, BoUnCyCaStLe? (or is this how we do it?)

Reply:
"rocksdb" is all lowercase, thats why I preferred lowercase bouncycastle.